lib/rucio/tests/test_replica.py

```diff
@@ -27,6 +27,7 @@
 from datetime import datetime, timedelta
 from json import dumps, loads
 
+import time
```

Comment:
sort alphabetically

Reply:
just tell me where you want it cause according to Martin it's not even in the right section
and of course also white lines matter